srcpkgs/duplicity/template

```diff
@@ -11,12 +11,23 @@ checkdepends="$depends python3-pytest python3-mock python3-pexpect
 short_desc="Encrypted bandwidth-efficient backup using the rsync algorithm"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-2.0-or-later"
-homepage="http://duplicity.nongnu.org/"
-changelog="http://duplicity.nongnu.org/vers8/CHANGELOG.md"
-distfiles="http://launchpad.net/duplicity/${version%.*}-series/${version}/+download/duplicity-${version}.tar.gz"
-checksum=bbe4679d37cc5523c2302d6d7933d32f365f7c9aea7ffc7827c45e3b7b2d2234
+homepage="https://duplicity.gitlab.io"
+changelog="${homepage}/vers8/CHANGELOG.html"
+distfiles="https://launchpad.net/duplicity/${version%.*}-series/${version}/+download/duplicity-${version}.tar.gz"
+checksum=b24aec0df4849a71de64b743fe9957f09bb80259898e455bdc612c78bdb374e2
 
-post_install() {
-	# drop doc
-	rm -rf ${DESTDIR}/usr/share/doc
+duplicity-doc_package() {
+	short_desc+=" - documentation"
+	depends="${sourcepkg}>=${version}_${revision}"
+	pkg_install() {
+		vmove usr/share/doc
+	}
+}
+
+duplicity-lang_package() {
```

Comment:
No, translations go to same package as binary using them.

If you do not like having translations to languages you do not know, set up noextract at `/etc/xbps.d/`:

    noextract=/usr/share/locale/??/LC_*
    noextract=/usr/share/locale/???/LC_*
    noextract=/usr/share/locale/??@*/LC_*
    noextract=/usr/share/locale/??_*/LC_*
    noextract=!/usr/share/locale/pl/LC_*
    noextract=!/usr/share/locale/pl_*/LC_*
    noextract=!/usr/share/locale/en/LC_*
    noextract=!/usr/share/locale/en_GB/LC_*
    noextract=!/usr/share/locale/en_US/LC_*